Makefile

```diff
@@ -25,8 +34,15 @@ localtest:
 install:
 	cp runc /usr/local/bin/runc
 
+build-deb:
+	./script/build-deb
+
+build-rpm:
+	./script/build-rpm
+
 clean:
-	rm runc
+	rm -rf bundles || true
+	rm runc || true
```

Comment:
With `-f`, when will `rm` fail?  Also, why not add `-f` to the second `rm` and skip the `|| true`?